src/strings/af.json

```diff
@@ -874,5 +874,6 @@
     "ForeignPartsOnly": "Slegs gedwonge/vreemde onderdele",
     "AiTranslated": "AI Vertaal",
     "MachineTranslated": "Masjien Vertaal",
-    "HearingImpairedShort": "HI/SDH"
-}
+    "HearingImpairedShort": "HI/SDH",
+    "LabelUnits": "(sekondes)"
+}
```

Comment:
Revert changes to languages other than `en-us`. Only Weblate should be used to work with other languages.